src/olympia/files/views.py

```diff
@@ -98,8 +97,7 @@ def browse(request, viewer, key=None, type='file'):
     data['poll_url'] = reverse('files.poll', args=[viewer.file.id])
     data['form'] = form
 
-    if (not waffle.switch_is_active('delay-file-viewer') and
-            not viewer.is_extracted()):
+    if not viewer.is_extracted():
         extract_file(viewer)
```

Comment:
I'm not sure I understand why it isn't delaying, but if this is what we want I guess that's cool. Why remove this from celery?